lesson08/assignment07/src/components/images-content.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+
+<!--image code -->
+  <div>
+    <img :src="image.src" :alt="image.alt" style="width:100%" id="noBorder" class="addBorder frame card-img-top img-fluid rounded">
+    <div class="caption">
+    <p>{{image.caption}}</p>
+    </div>
+  </div>
+  
+</template>
+
+<script>
+
+//props
+export default {
+    name: 'photos',
+    props: {
```

Comment:
So, you are validating that you receive an image object, but not that it must contain alt : string, caption : string, src : string.  AS is you could pass in an empty object and it would break the site, but still pass validation.